static/system.json

```
@@ -940,6 +940,130 @@
                 "pfs-season-3-bestiary",
                 "pfs-season-4-bestiary"
             ]
+        },
+        {
+            "name": "Pathfinder Adventures",
+            "sorting": "m",
+            "packs": [
+                "fall-of-plaguestone-bestiary",
+                "malevolence-bestiary",
+                "menace-under-otari-bestiary",
+                "one-shot-bestiary",
+                "shadows-at-sundown-bestiary",
+                "the-enmity-cycle-bestiary",
+                "the-slithering-bestiary",
+                "troubles-in-otari-bestiary",
+                "night-of-the-gray-death-bestiary",
+                "crown-of-the-kobold-king-bestiary"
+            ]
+        },
+        {
+            "name": "Pathfinder Adventure Paths",
+            "sorting": "m",
+            "packs": [
+                "abomination-vaults-bestiary",
+                "age-of-ashes-bestiary",
+                "agents-of-edgewatch-bestiary",
+                "blood-lords-bestiary",
+                "extinction-curse-bestiary",
+                "fists-of-the-ruby-phoenix-bestiary",
+                "gatewalkers-bestiary",
+                "outlaws-of-alkenstar-bestiary",
+                "kingmaker-bestiary",
+                "quest-for-the-frozen-flame-bestiary",
+                "strength-of-thousands-bestiary",
+                "stolen-fate-bestiary"
+            ]
+        },
+        {
+            "name": "Pathfinder Rulebook Bestiaries",
```

```suggestion
            "name": "Rulebook Bestiaries",
```
Maybe split between "Rulebook" and "Lost Omens"?

---

My only issue there is that for most people they don't care about the distinction between rulebooks and lost omens books.

This also has the blog stuff in there, so maybe we just call it Bestiaries?

---

That sounds good to me